### PR TITLE
Fixes welcome notice styles for 5.2, adds screenshots and real links.

### PR DESCRIPTION
**Pull request by dereksmart**
Completes the unchecked to-do's in #7531

- Adds a screenshot of the CF in the message 
- "Try it now!" button will take you to `wp-admin/post-new.php` to create a new post
- Fixed the font styles to match the mock a little better
- Updated the svg, it was not showing outlines before.  

screen shot 2017-07-28 at 10 40 51 am

### CODE REVIEW

**Comment by keoshi** [comment]
Looks good but I worry if that's too much text for a quick “what's new” glimpse. I'd also try to add more a bit margin between the image and text around it. Otherwise, looks gooood! ⭐️

**Comment by ashleighaxios** [comment]
To suggest shorter copy that's a bit more active:
> Get the newest release to use the updated contact form. With this update you can add the contact form from a new button on the right side of your editor's toolbar and see a preview that can be adjusted inside the editor.
[ Image ] 
> Once the contact form is added, you can customize the fields to fit your needs by tapping the preview or clicking the edit button. 
[ try it out button ]
_Read the full release post_

If the release post doesn't have a video or animated GIF from @MichaelArestad, we should add it. It might be nice for those on the fence see a motion demo of what the process looks like.

**Comment by eliorivero** [comment]
At this stage the text can't be modified. The strings have already been sent for translation.
I'll add some spacing around the text.

**Comment by eliorivero** [comment]
Updated spacing:

### Before
captura de pantalla 2017-07-28 a las 13 02 37

### After
captura de pantalla 2017-07-28 a las 13 02 20

**Comment by jeffgolenski** [comment]
@dereksmart @eliorivero I just optimized the contact form screenshot. It was originally at 60kb... I got it down to 28kb. Give me a shout whenever images are added to the plugin so I can do this... or download the ImageOptim app for your mac if you wanna do it yourself before merging. 

🙇

File: _inc/client/components/jetpack-dialogue/style.scss [hunk 1]
```diff
@@ -61,3 +75,7 @@
 	top: rem( 100px );
 	opacity: .90;
 }
+
+.jp-dialogue__cta-container {
+	padding: rem( 8px ) 0 0 0;
```
Comment (jeffgolenski) [inline _inc/client/components/jetpack-dialogue/style.scss]: One too many `0` here. We can make this shorter by writing `padding: rem( 8px ) 0 0;`

File: _inc/client/components/jetpack-dialogue/style.scss [hunk 2]
```diff
@@ -15,10 +15,24 @@
...
+		font-size: rem( 14px );
+		color: $gray-text-min;
+	}
+
+	img {
+		max-width: 100%;
+	}
+
+	p + img {
+		margin: 1em 0 .5em;
```
Comment (jeffgolenski) [inline _inc/client/components/jetpack-dialogue/style.scss]: Make sure we're consistently using `rem` and not ems.

**Comment by dereksmart** [comment]
thanks @jeffgolenski!

**Comment by eliorivero** [comment]
cherry picked to 5.2 in db50925687a0c98658525e053063faef339765e8

**Pull request merged**